src/components/branded-card/index.tsx

```diff
@@ -4,7 +4,7 @@
  */
 
 import { CSSProperties } from 'react'
-import Image from 'next/legacy/image'
+import Image, { type StaticImageData } from 'next/legacy/image'
```

Comment:
Previously, `nextOptimizedImages` returned a string for these imports. Now that this is handled by Next.js natively, these imports return `StaticImageData`.